Source/Core/Core/IOS/VersionInfo.h

```diff
@@ -37,5 +37,30 @@ struct MemoryValues
 };
 
 const std::array<MemoryValues, 41>& GetMemoryValues();
+
+enum Feature
```

Comment:
Changed it to be an enum class. I ended up only implementing operator| and operator|=, since we weren't going to be using the rest.